tests/global.rs

```diff
@@ -0,0 +1,27 @@
+use accounting_allocator::{AccountingAlloc, AllTimeAllocStats};
+
+use crossbeam_utils::thread::scope;
+
+#[global_allocator]
+static GLOBAL_ALLOCATOR: AccountingAlloc = AccountingAlloc::new();
+
+#[test]
+fn global_display() {
+    scope(|scope| (0..9).for_each(|idx| drop(scope.spawn(move |_| drop(Vec::<u8>::with_capacity(10000 * idx))))))
+        .unwrap();
+
+    println!("{GLOBAL_ALLOCATOR:?}");
+    println!("{GLOBAL_ALLOCATOR}");
+    println!("{GLOBAL_ALLOCATOR:?}");
+}
```

Comment:
Should these have at least minimal assertions?

Reply:
Hmm, I didn't want to rely on parsing display/debug output, so I was mostly eyeballing this during development. It's hard to pin down exactly what will be outputted, since Rust is allowed to (and does) allocate whatever it needs using the global allocator internally.

These tests are mostly to see that the display / debug implementations work and don't infinitely recurse or panic or abort. We do get some implicit assertions from the system malloc on most hosts for double-free / leak detection (and could probably add a memory sanitizer on top of that later), as well as the panics inside our AccountingAlloc code.

I'll add all that in a comment explanation!

Reply:
I was thinking you could at least lower-bound all the fields, or even just check that they're non-zero. That demonstrates that the allocator was used in an automated way. But checking the recursion is pretty important too!

Reply:
fixed in ff68b41